src/CommunicationApi/Controllers/ContentController.cs

```diff
@@ -43,7 +43,11 @@ public ContentController(ILogger<ContentController> logger, IEnumerable< IMediaS
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         public async Task<IActionResult> GetMessages([FromQuery] string boxId)
         {
+            _logger.LogEvent($"Getting message for box {boxId}");
```

Comment:
Let's use a trace here with structured logging
```suggestion
            _logger.LogInformation("Getting message for box {BoxId}", boxId);
```